docs/modules/deployment/pages/core/system-requirements.adoc

```diff
@@ -53,19 +53,19 @@ endif::[]
 | {compatible-rocky-linux}
 |===
 
-NOTE: See the xref:releasenotes:whatsnew.adoc[release notes] for information about platform support changes, including deprecated platforms.
+See the xref:releasenotes:whatsnew.adoc[release notes] for information about platform support changes, including deprecated platforms.
 
 == Compatibility
 
 {page-component-title} {full-display-version} is compatible with the following components.
-Components marked with an asterisk are required.
+Components marked with an asterisk are required:
 
 [options="autowidth"]
 |===
 | Component | Version Compatibility
 
-| OpenNMS Helm
-| {compatible-opennms-helm}
```

Comment:
Is this saying that the plugin is compatible with... itself? Or should this really be saying what version of _Grafana_ it's compatible with?

(regardless, we should probably make this new release say `9+` rather than `7+`)

Reply:
The table is saying that Horizon X is compatible with the following components, one of which is the product formerly known as Helm, and its version. Yes, should be updated to 9, if it isn't already.

Reply:
Oh, duh! I thought this was in the `opennms-helm` repo, not the `opennms` repo!

Still, 7 is technically not wrong, I guess, the old grafana 7 stuff will still work with it.